### DISCUSSION TITLE
Move /NMS to Separate Drive

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5703] [Unsupported Versions] [answered]
Hello all,

I am considering moving the /NMS directory structure to its own physical drive. Has anyone done this? Any pitfalls to doing this? Anything else I should move also?

Thanks,
Steve

**Comment by clairmont32** [comment, marked as answer]
Yes, the docs cover some methods for doing this. You may want to make it a LVM so you can continue expanding it as needed.

https://docs.securityonion.net/en/latest/new-disk.html

**Reply by stevefxp1** [reply]
That makes sense...but the question becomes can I edit the install script, so I can point it to install /NMS and accompanying directories to my new drive? Ideally I would like to use LVM but since this is a second drive it does not get engaged at Ubuntu install time.

**Reply by n0th1nt0lOOz** [reply]
I think you can link/mount it to second drive. In my case, I use a NAS with a ISCSI to mount the nsm folder to NAS

**Reply by clairmont32** [reply]
@stevefxp1 no, this is a post install item. You can delete /nsm directly after install and mount the 2nd drive in afterwards.

**Reply by dougburks** [reply]
@stevefxp1 No need to edit the install script. After you've completed the Ubuntu installation but before you start our installer, you can manually configure `/nsm` however you want to (including LVM). Then when you start our installer, it will then use your `/nsm` however you've configured it.
